src/app/models/form.server.model.ts

```diff
@@ -184,6 +186,7 @@ const compileFormModel = (db: Mongoose): IFormModel => {
               type: Boolean,
               default: false,
             },
+            _id: false,
```

Comment:
has mongoose been populating an `_id` field automatically?

Reply:
i'm wondering if there's any benefit to keeping that, in case we need to have an audit trail in the future

Reply:
Yea, mongoose has been populating it. Also can la, this one your call

Reply:
let's keep it